index.html

```diff
@@ -15,10 +15,25 @@
   </head>
   <body>
     <div class="container">
+      <div class="show-hamburger hidden">
+        <div class="cross-container">
+          <img
+            src="./assets/icons/cancelHamburger.png"
+            alt=""
+            class="close-cross"
+          />
+        </div>
+        <ul class="show-links-hamburger">
+          <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
+          <li><a href="#about-info" class="menu-links">About</a></li>
```

Comment:
- [ ] For ease of access and smooth navigation experience to users or visitors, I think you should consider redirecting the to your `About me` section whenever they click on this link. The link is currently referencing the first page/landing page when what I wanted to navigate to was your About me section, just before the `Contact section`. Please consider removing the `id` from the first page and add it to the section where you have this view :point_down: 
![image](https://user-images.githubusercontent.com/99715640/223211141-e5918305-9496-4d3c-ba09-de12bb27d81b.png)